src/Version.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Version;
+
+class Version
+{
+    public const string SCRIPTS_DIRECTORY = "src/scripts/";
+
+    public function __construct(
+        public bool $long = false,
+    ) {}
+
+    public function setLong(bool $long): void
+    {
+        $this->long = $long;
+    }
+
+    public function generate(): string
+    {
+        return shell_exec(self::SCRIPTS_DIRECTORY . "check.sh")
```

Comment:
I would like to use Symfony Process class instead of this `shell_exec()`.
It provides extra level of security. It is also very convenient because of an object-oriented interface. 
https://symfony.com/doc/current/components/process.html

Reply:
Its very simple shell command, can we stay with builtin command in PHP language?

Reply:
changed